packages/client/jwtsecret

```diff
@@ -0,0 +1 @@
+aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

Comment:
Did you mean to include this file?

Reply:
Ugh this (plus genesis) is from running docker in a `ADD` mode instead of `COPY`. Will remove